ansible/config_sonic_basedon_testbed.yml

```diff
@@ -112,7 +112,7 @@
       port_alias: "{{ port_alias }}"
       vlan_intfs: "{{ vlan_intfs }}"
     delegate_to: localhost
-    when: "'dualtor' in topo"
+    when: "'dualtor' in topo or 'cable' in topo"
```

Comment:
Should we check for `'cable'` or `'cable-test'`? I feel that if we need to develop more than one cable testing topology, we will simply add suffixes (e.g., `cable-test-xxx`, `cable-test-yyy`, etc.). Might be better to check for the longer substring.